### PR TITLE
runner --cloud-region: GCP zones

### PR DESCRIPTION
**Pull request by casperdcl**
- mention GCP needs zones not regions
- sync self-hosted docs with cmd ref
- add some links

### CODE REVIEW

File: content/docs/ref/runner.md [hunk 1]
```diff
@@ -36,8 +36,8 @@ Any [generic option](/doc/ref) in addition to:
   *current price*].
 - `--cloud-region={us-west,us-east,eu-west,eu-north,...}`:
   [Region](https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task#cloud-regions)
-  where the instance is deployed. Also accepts native cloud regions [default:
-  `us-west`].
+  where the instance is deployed. Also accepts native AWS/Azure region or GCP
+  zone [default: `us-west`].
```
Comment (0x2b3bfa0) [inline content/docs/ref/runner.md:40]:
```suggestion
  where the instance is deployed. Also accepts native AWS/Azure regions or GCP
  zones [default: `us-west`].
```

Plural, perhaps? 🤔

Comment (casperdcl) [reply content/docs/ref/runner.md:40]: Can you specify multiple in one go?

Comment (0x2b3bfa0) [reply content/docs/ref/runner.md:40]: No

File: content/docs/self-hosted-runners.md [hunk 2]
```diff
@@ -164,9 +164,15 @@ The `cml runner` command supports many options (see the
 - `--cloud-spot-price=<...>`: Maximum spot instance USD bidding price.
 - `--cloud-region={us-west,us-east,eu-west,eu-north,...}`:
   [Region](https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task#cloud-regions)
-  where the instance is deployed. Also accepts native cloud regions.
-- `--cloud-permission-set=<...>`: AWS instance profile or GCP instance service
-  account.
+  where the instance is deployed. Also accepts native AWS/Azure region or GCP
+  zone.
```
Comment (0x2b3bfa0) [inline content/docs/self-hosted-runners.md:168]:
```suggestion
  where the instance is deployed. Also accepts native AWS/Azure regions or GCP
  zones.
```

Same as https://github.com/iterative/cml.dev/pull/194/files#r799866101

File: content/docs/ref/runner.md (hunk 1 above)
Comment (0x2b3bfa0) [reply content/docs/ref/runner.md:40]: Still, plural feels more natural to my mind. Perhaps just a non–native brain artifact.

**Review by 0x2b3bfa0 — APPROVED** [review]

File: content/docs/ref/runner.md (hunk 1 above)
Comment (casperdcl) [reply content/docs/ref/runner.md:40]: both are grammaticisionally correk. But singular is more precise for this `--flag`.

**Pull request merged**